arcgis-android-toolkit/src/main/res/layout/item_bookmark_row.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2019 Esri
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should be a bindable layout.

Reply:
can you explain why ? the binding is working in the `BookmarkAdapter.ViewHolder.bind()` method without this being a bindable layout..

Reply:
But this isn't true data binding, explained here: https://github.com/Esri/arcgis-runtime-toolkit-android/pull/146/files/e451f2f85f26b6f0a9db0d8122a647217b3911e1#diff-3852308ac2ab80c12bbc863fe4507601R72